src/mlpack/tests/io_test.cpp

```diff
@@ -1024,14 +1049,15 @@ BOOST_AUTO_TEST_CASE(RawIntegralParameter)
   IO::GetRawParam<double>("double") = 3.0;
 
   // Now when we get it, it should be what we just set it to.
-  BOOST_REQUIRE_CLOSE(IO::GetParam<double>("double"), 3.0, 1e-5);
+  REQUIRE(IO::GetParam<double>("double") == Approx(3.0).epsilon(1e-7));
 }
 
 /**
  * Test that we can load a dataset with a pre-set mapping through
  * IO::GetRawParam().
  */
-BOOST_AUTO_TEST_CASE(RawDatasetInfoLoadParameter)
+TEST_CASE_METHOD(IOTestDestroyer, "RawDatasetInfoLoadParameter",
+                "[IOTest]")
```

Comment:
```suggestion
TEST_CASE_METHOD(IOTestDestroyer, "RawDatasetInfoLoadParameter", "[IOTest]")
```

Looks like this would fit into the same line.